docs/process.md

```diff
@@ -113,6 +136,7 @@ If the PR was from a contributor, you can merge it at this point.
 
 - Put completed issues under current milestone
 - Remove milestones for issues which weren't finished
+- Determine if open PRs can be merged by the next milestone, and close them otherwise
```

Comment:
More specifically, (a) add a comment in open PRs to close in the next milestone (b) close PRs that have not been making progress in spite of reminders.